tests/__snapshots__/InlineFunctions.test.js.snap

```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InlineFunctions.vue Functions kept 1`] = `
+<div>
+  <h1 title="function () {
+        return true;
+      }">Classic</h1>
+  <h2 title="function () {
+        return true;
+      }">Multi-line</h2>
+  <h3 title="function (x) {
+        return !x;
+      }">Arrow</h3>
```

Comment:
@cameron-martin You can see here what Vue renders the templates out to. the One liner, Multi-line, and Arrow function, are all output as the same multi-line approach.

Reply:
Interesting. How does that transformation get peformed?

Reply:
Not sure when or how, but by the time we are checking for it, they are all the same.